contracts/test/emergency/StarkNetValidator.test.ts

```diff
@@ -16,19 +16,20 @@ import { deployMockContract, MockContract } from '@ethereum-waffle/mock-contract
 import {
   account,
   addCompilationToNetwork,
-  startNetwork,
-  startNode,
-  IntegratedDevnet,
+  loadConfig,
+  NetworkManager,
+  FunderOptions,
+  Funder,
 } from '@chainlink/starknet'
 
 describe('StarkNetValidator', () => {
+  const config = loadConfig()
+  const optsConf = { config, required: ['devnet', 'hardhat'] }
```

Comment:
This test shouldn't know much about the environment, the specifics of the networks it requires.

Instead, we should request network/s of specific blockchain type, and then the test gets what's configured in the env (depends how we run the test). It can be `devnet + hardhat` or for example `goerli + goerli_starknet', but we request for:
```suggestion
  const optsConf = { config, required: ['starknet', 'ethereum'] }
```